apache2/apache2_util.c

```diff
@@ -271,24 +271,23 @@ static void get_short_filename(char* waf_filename) {
 }
 
 /**
- * get crs type and version.
+ * get crs type and version, format will be changed in waf_log_util.cc parse_rule_set_type/parse_rule_set_version
  */
-static void get_ruleset_type_version(char* waf_ruleset_info, char* waf_ruleset_type, char* waf_ruleset_version) {
-    char ruleset_info_no_quote[200] = "";
+static void get_ruleset_type_version(const char* waf_ruleset_info, char* waf_ruleset_type, char* waf_ruleset_version) {
     char *type_start = NULL;
     char *type_end = NULL;
 
-    get_field_value("\"", "\"", waf_ruleset_info, ruleset_info_no_quote);
-    type_start = ruleset_info_no_quote;
+    type_start = waf_ruleset_info;
```

Comment:
minor - you can check the waf_ruleset_info before assignment

Reply:
check none in the next line